### PR TITLE
INST-21855-deploy-oneAPI-2022.1.2-to-Spack

### PR DESCRIPTION
**Pull request by VladimirUspenskii**
Add oneAPI packages from the 2022.1.2 release
oneapi-test CI results: https://github.com/VladimirUspenskii/oneapi-spack-tests/actions/runs/1789365251

### CODE REVIEW

**Review by rscohn2 — DISMISSED** [review]
Looks good.

File: var/spack/repos/builtin/packages/intel-oneapi-ccl/package.py [hunk 1]
```diff
@@ -18,6 +18,10 @@ class IntelOneapiCcl(IntelOneApiLibraryPackage):
     depends_on('intel-oneapi-mpi')
 
     if platform.system() == 'Linux':
+        version('2021.5.1',
+                url='https://registrationcenter-download.intel.com/akdlm/irc_nas/18472/l_oneapi_ccl_p_2021.5.1.494_offline.sh',
+                sha256='237f45d3c43447460e36eb7d68ae3bf611aa282015e57c7fe06c2004d368a68e',
+                expand=False)
```
Comment (rscohn2) [inline var/spack/repos/builtin/packages/intel-oneapi-ccl/package.py:25]:
```suggestion
                expand=False)
                
```

add blank line to change hash and trigger gitlab retest

**Comment by rscohn2** [comment]
@alalazo : Could you merge this one, too?

**Review by alalazo — APPROVED** [review]
Maintainer approved

**Pull request merged**